.github/workflows/python.yml

```diff
@@ -106,3 +106,23 @@ jobs:
       run: bash scripts/deploy_docker.sh ${GITHUB_REF##*/}
       env:
         DOCKERHUB_PROJECT: safe-transaction-service
+  autodeploy:
+    runs-on: ubuntu-latest
+    needs: [test-app, docker-deploy]
```

Comment:
Only `docker-deploy` is needed, as it depends on `test-app`

Reply:
got it